flask_weather_app.py

```diff
@@ -0,0 +1,33 @@
+from flask import Flask, render_template, request
+import requests
+
+app = Flask(__name__)
+
+# Assuming you have your openweather_api_key in a config module
+from config import openweather_api_key as api_key
+
+def get_weather(city_name, forecast=False):
+    base_url = "http://api.openweathermap.org/data/2.5/weather" if not forecast else "http://api.openweathermap.org/data/2.5/forecast"
```

Comment:
use multiline if/else (no ternary) and abstract repeated base url in constant at top of the module.